hrm-domain/hrm-service/service-tests/case/caseValidation.ts

```diff
@@ -28,42 +28,33 @@ export const validateCaseListResponse = (
 ) => {
   expect(actual.status).toBe(200);
   if (count === 0) {
```

Comment:
I think we still need to validate the `count` property - what is the thinking behind removing those checks?

Reply:
I made a commit I did not mean to. with this change. Reverting it...